lib/iris/tests/test_cartography.py

```diff
@@ -4,26 +4,27 @@
 # See LICENSE in the root of the repository for full licensing details.
```

Comment:
NTS:
I ran out of time when reviewing this, and didn't have time to check that everything on the checklist was accounted for. _**All files need to be ran through the checklist before merge!**_